### ISSUE TITLE
Issues building rat database

### ISSUE THREAD

**Issue by gavinkurgan** [charite/jannovar#441]
Hello! 

I appear to be having some issues building databases for use with Jannovar from RefSeq for Rat (Rattus norvegicus RN6). When running the following command:
`jannovar download -s my_refseq.ini -d rn6/refseq`

I get the following error:
```
... 
INFO Building ReferenceDictionary...
INFO Parsing transcripts...
INFO Loading feature records
INFO Determined GTF/GFF file version to be GFF3
Exception in thread "main" java.lang.NullPointerException
        at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:782)
        at com.google.common.base.Splitter.split(Splitter.java:376)
        at de.charite.compbio.jannovar.impl.parse.refseq.RefSeqParser.parseGeneID(RefSeqParser.java:323)
        at de.charite.compbio.jannovar.impl.parse.refseq.RefSeqParser.createNewTranscriptModelBuilder(RefSeqParser.java:310)
        at de.charite.compbio.jannovar.impl.parse.refseq.RefSeqParser.loadTranscriptModels(RefSeqParser.java:242)
        at de.charite.compbio.jannovar.impl.parse.refseq.RefSeqParser.run(RefSeqParser.java:95)
        at de.charite.compbio.jannovar.datasource.RefSeqJannovarDataFactory.parseTranscripts(RefSeqJannovarDataFactory.java:36)
        at de.charite.compbio.jannovar.datasource.JannovarDataFactory.build(JannovarDataFactory.java:105)
        at de.charite.compbio.jannovar.cmd.download.DownloadCommand.run(DownloadCommand.java:43)
        at de.charite.compbio.jannovar.Jannovar.main(Jannovar.java:64)
```

It seems like it might be related to aliases, but I can't tell if it is a bug or if I am potentially missing something. Here is the config info I used in the .ini file:
```
[rn6/refseq]
type=refseq
alias=MT,M,chrM
chromInfo=http://hgdownload.soe.ucsc.edu/goldenPath/rn6/database/chromInfo.txt.gz
chrToAccessions=https://ftp.ncbi.nlm.nih.gov/genomes/R_norvegicus/ARCHIVE/ANNOTATION_RELEASE.105/Assembled_chromosomes/chr_accessions_Rnor_6.0
chrToAccessions.format=chr_accessions
gff=https://ftp.ncbi.nlm.nih.gov/genomes/R_norvegicus/ARCHIVE/ANNOTATION_RELEASE.105/GFF/ref_Rnor_6.0_top_level.gff3.gz
rna=https://ftp.ncbi.nlm.nih.gov/genomes/R_norvegicus/ARCHIVE/ANNOTATION_RELEASE.105/RNA/rna.fa.gz
``` 
Any light you could shed on this issue would be much appreciated!

Thanks,
Gavin

**Comment by gavinkurgan** [comment]
So as a follow up to this it appears that there was possibly a problematic name in the uncurated transcripts from refseq for rat (or maybe jannovar had an issue handling it?). I was able to complete the previous command simply by adding the following to the config:
```
onlyCurated=True
```

It is still not clear to me if this was due to an issue with jannovar or the input files from NCBI. If it is due to the input files, it might be useful to add some error handling for this issue.
